src/main/kotlin/no/nav/familie/ef/iverksett/iverksetting/IverksettingService.kt

```diff
@@ -72,6 +73,7 @@ class IverksettingService(val taskRepository: TaskRepository,
     }
 
     private fun førstePubliseringsflytTask(iverksett: Iverksett) = when {
+        iverksett.erGOmregning() -> VedtakstatistikkTask.TYPE
```

Comment:
Du kan legge inn endringen i `Task.opprettNestePubliseringTask` for å ha "samme flyt". For `
SendPerioderTilInfotrygdTask` vil ikke gjøre noe for g-omregninger siden det alltid finnes en tidligere behandling. 
```
fun Task.opprettNestePubliseringTask(erMigrering: Boolean = false): Task {
    val nesteTask = if ((erMigrering || erGOmregning) && this.type == SendPerioderTilInfotrygdTask.TYPE) {
```

Reply:
Da må vi legge til en parameter til funksjonen opprettNestePubliseringTask. Da ender vi opp med to boolske verdier som gjør det samme. Det virker litt hårete. Er ike så sikker på at det er noen grunn til å sende dem til infotrygd heller.

Reply:
Forsåvidt sant, men bare opplever at vi har litt sjekk her og litt sjekk der nå, men ikke så stor forskjell kanskje.
Disse vil aldri sendes til infotrygd siden de har en behandling fra før (g-omregning blir aldri første behandlingen vår)

Reply:
Vi burde vel kanskje kun gjøre dette når det gjelder maskinelle g-omregninger, mens hvis det er manuell - som vi ikke helt har kontroll på, så burde vi kanskje kjøre hele løypa?

Reply:
Manuell g-omregning burde låses for andre endringer enn inntekt og samordningsfradrag.

Reply:
Jeg synes endringen er fin nå som vi vet at infotrygd heller ikke trenger melding.
Tenker Johan har et godt poeng angående maskinell vs manuell, siden det ikke er lagt inn noen form for sperrer i frontend på hva som kan endres ved g-omregning. Dersom det endres til sjekk på erMaskinellGOmregning i stedet, så tenker jeg det er greit. Det blir noen meldinger ekstra på de manuelle der det kun er inntekt og samordningsfradrag som endres, men det tåler konsumentene.

Reply:
> Manuell g-omregning burde låses for andre endringer enn inntekt og samordningsfradrag.

Johan lager et forslag til validering